app/[locale]/(authenticated)/profile/page.tsx

```diff
@@ -1,12 +1,114 @@
-import { query } from "@/app/db"
-import { verifySession } from "@/app/session"
+import FormLabel from "@/app/components/FormLabel";
+import TextInput from "@/app/components/TextInput";
+import ViewTitle from "@/app/components/ViewTitle";
+import { query } from "@/app/db";
+import { verifySession } from "@/app/session";
+import { ResolvingMetadata, Metadata } from "next";
+import { getTranslations } from "next-intl/server";
+import FieldError from "@/app/components/FieldError";
+import Button from "@/app/components/Button";
+import ProfileForm from "./ProfileForm";
+
+export async function generateMetadata(
+  _: any,
+  parent: ResolvingMetadata
+): Promise<Metadata> {
+  const t = await getTranslations("ProfileView");
+  const { title } = await parent;
+
+  return {
+    title: `${t("title")} | ${title?.absolute}`,
+  };
+}
 
 export default async function ProfileView() {
-    const session = await verifySession()
-    const result = session ? await query<{ name?: string, email: string }>(`SELECT name, email FROM "User" WHERE id = $1`, [session.user.id]) : undefined
-    const user = result?.rows[0]
+  const session = await verifySession();
+  const result = session
+    ? await query<{ name?: string; email: string }>(
```

Comment:
If there is no session, we should use `notFound` from the nextjs imports. No reason to load the profile view if there is no user to show